src/bootstrap/dist.rs

```diff
@@ -1245,35 +1254,38 @@ impl Step for Extended {
         }
         rtf.push_str("}");
 
+        fn filter(contents: &str, marker: &str) -> String {
+            let start = format!("{}-start", marker);
+            let end = format!("{}-end", marker);
```

Comment:
Let's have something like `tool-rls-start` instead of just `rls-start` to avoid the unfortunate (and hard to debug)` situation with two tools that have the same suffix conflicting here.